### ISSUE TITLE
Incorrect display of armor values in shops

### ISSUE THREAD

**Issue by x-qq** [calref/cboe#265]
Information currently provided by armor vendors about how much damage a piece armor mitigates does not match reality.

Example: bronze plate armor (level 9, protection 1, bonus 0) in bladbase armor shop:

![Untitled](https://user-images.githubusercontent.com/55463047/74656893-c2e63600-5197-11ea-94ab-28a3bf9ca339.png)

The shop claims that it blocks flat 10 damage.

However in reality, without factoring in Defense skill, it blocks:
1d(level+protection) + floor(bonus/2) + 1d(bonus), so 1d10 = 5.5 damage on average.

**Comment by x-qq** [comment]
I have also just noticed that in the item display window on the above screenshot, it shows "bonus 1" when in fact this armor has no bonus according to items.xml https://github.com/calref/cboe/blob/master/rsrc/bases/bladbase/items.xml#L2421

**Comment by CelticMinstrel** [comment]
I was about to say that this matches the display in the original game, but then I realized it says "10-10" which seems odd. Well, either way, it can certainly be changed to make more sense.

**Comment by NQNStudios** [comment]
Is this related to 68ef066dca17823ddf9b99f2de9c6e0902b0d6bf?

**Comment by CelticMinstrel** [comment]
Yes, presumably that fixes it, but I haven't looked closely. Feel free to close if shops look correct now.
